### PR TITLE
Api instance methods

### PR DESCRIPTION
**Pull request by Tintef**
- https://trello.com/c/5lwbdf2w/101-api-indexmd-instance-methods-data
- https://trello.com/c/VsTCyPWS/102-api-indexmd-instance-methods-events
- https://trello.com/c/N5VrSZf9/103-api-indexmd-instance-methods-lifecycle

Note: I don't really the translation for `event listener(s)` I used, I would love any suggestions on that!

### CODE REVIEW

File: src/v2/api/index.md [hunk 1]
```diff
@@ -1462,115 +1462,115 @@ type: api
...
-  // later, teardown the watcher
+  // luego, destruir el `watcher`
   unwatch()
   ```
 
-- **Option: deep**
+- **Opción: deep**
 
-  To also detect nested value changes inside Objects, you need to pass in `deep: true` in the options argument. Note that you don't need to do so to listen for Array mutations.
+  Para también detectar valores anidados dentro de Objetos, usted debe pasar `deep: true` en las opcoines. Note que usted no necesita realizar esto para mutaciones de Arreglos.
```
Comment (FLarra) [inline src/v2/api/index.md]: Minor typo here:
```suggestion
  Para también detectar valores anidados dentro de Objetos, usted debe pasar `deep: true` en las opciones. Note que usted no necesita realizar esto para mutaciones de Arreglos.
```

File: src/v2/api/index.md [hunk 2]
```diff
@@ -1462,115 +1462,115 @@ type: api
...
 
   ``` js
   vm.$watch('someObject', callback, {
     deep: true
   })
   vm.someObject.nestedValue = 123
-  // callback is fired
+  // se dispará la función `callback`
```
Comment (FLarra) [inline src/v2/api/index.md]: I think you meant:

```suggestion
  // se disparará la función `callback`
```

File: src/v2/api/index.md [hunk 3]
```diff
@@ -1462,115 +1462,115 @@ type: api
...
 
-  Passing in `immediate: true` in the option will trigger the callback immediately with the current value of the expression:
+  Pasar `immediate: true` en las opciones disparará la función _callback_ inmediatamente con el nuevo valor de la expresión:
 
   ``` js
   vm.$watch('a', callback, {
     immediate: true
   })
-  // `callback` is fired immediately with current value of `a`
+  // se dispará el `callback` inmediatamente con el nuevo valor de `a`
```
Comment (FLarra) [inline src/v2/api/index.md]: Same misspelling here:

```suggestion
  // se disparará el `callback` inmediatamente con el nuevo valor de `a`
```

File: src/v2/api/index.md [hunk 4]
```diff
@@ -1582,41 +1582,41 @@ type: api
...
 
-- **Arguments:**
+- **Argumentos:**
   - `{string} event`
   - `{Function} callback`
 
 - **Uso:**
 
-  Listen for a custom event, but only once. The listener will be removed once it triggers for the first time.
+  Escucho un evento personalizado, pero solamente una vez. La escucha será removida una vez que es disparada por primera vez.
```
Comment (FLarra) [inline src/v2/api/index.md]: I think it's better to change this for:

```suggestion
  Escucha por un evento personalizado, pero solamente una vez. El _listener_ será removido una vez que es disparado por primera vez.
```

What do you think?

File: src/v2/api/index.md [hunk 5]
```diff
@@ -1582,41 +1582,41 @@ type: api
...
-- **Arguments:**
-  - `{string | Array<string>} event` (array only supported in 2.2.2+)
+- **Argumentos:**
+  - `{string | Array<string>} event` (arreglo solamente es soportado en 2.2.2+)
   - `{Function} [callback]`
 
 - **Uso:**
 
-  Remove custom event listener(s).
+  Remover escuchas de evento(s) personalizado(s).
```
Comment (FLarra) [inline src/v2/api/index.md]: I agree with you it's difficult to translate this in a way it sounds natural. Maybe what we can do is translate it as "escuchas" but add `(_listener_)` next to it? 🤔

File: src/v2/api/index.md [hunk 6]
```diff
@@ -1582,41 +1582,41 @@ type: api
...
 
-  - If no arguments are provided, remove all event listeners;
+  - Si ningún argumento es provisto, remueve todas las escuchas a eventos.
```
Comment (FLarra) [inline src/v2/api/index.md]: Maybe it's better:

```suggestion
  - Si ningún argumento es proporcionado, remueve todas las escuchas a eventos.
```

what do you think?

File: src/v2/api/index.md [hunk 7]
```diff
@@ -1739,99 +1739,99 @@ type: api
...
-- **Returns:** `vm` - the instance itself
+- **Retorna:** `vm` - la propia instancia
 
 - **Uso:**
 
-  If a Vue instance didn't receive the `el` option at instantiation, it will be in "unmounted" state, without an associated DOM element. `vm.$mount()` can be used to manually start the mounting of an unmounted Vue instance.
+  Si la instancia de Vue no recibió la opción `el` durante la instanciación, ella estará en el estado "no montada" sin un elemento del DOM asociado. `vm.$mount()` puede ser utilizado para comenzar a montar una instancia de Vue no montada.
 
-  If `elementOrSelector` argument is not provided, the template will be rendered as an off-document element, and you will have to use native DOM API to insert it into the document yourself.
+  Si el argumento `elementOrSelector` no es provisto, la plantilla será renderizara como un elemento "fuera del documento", y ustede deberá utilizar la API nativa del DOM para insertarla en el documento.
```
Comment (FLarra) [inline src/v2/api/index.md]: Typo:

```suggestion
  Si el argumento `elementOrSelector` no es provisto, la plantilla será renderizará como un elemento "fuera del documento", y usted deberá utilizar la API nativa del DOM para insertarla en el documento.
```

File: src/v2/api/index.md [hunk 8]
```diff
@@ -1739,99 +1739,99 @@ type: api
...
 
-  The method returns the instance itself so you can chain other instance methods after it.
+  El método retorna la propia instancia para que usted pueda encadenar otros métodos de instacia luego.
```
Comment (FLarra) [inline src/v2/api/index.md]: Typo here:

```suggestion
  El método retorna la propia instancia para que usted pueda encadenar otros métodos de instancia luego.
```

File: src/v2/api/index.md [hunk 9]
```diff
@@ -1739,99 +1739,99 @@ type: api
...
+- **Vea también:**
+  - [Diagrama de Ciclo de Vida](../guide/instance.html#Diagrama-del-Ciclo-de-vida)
+  - [Renderizado del lado de Servidor](../guide/ssr.html)
 
 ### vm.$forceUpdate()
 
 - **Uso:**
 
-  Force the Vue instance to re-render. Note it does not affect all child components, only the instance itself and child components with inserted slot content.
+  Fuerza a la instancia de Vue a renderizarse nuevamente. Note que esto no afecta a todos sus componentes hijo, solamente a la propia instancia y componentes hijos con contenido insertado en _slot_.
```
Comment (FLarra) [inline src/v2/api/index.md]: Maybe "hijo" should be in plural here?

```suggestion
  Fuerza a la instancia de Vue a renderizarse nuevamente. Note que esto no afecta a todos sus componentes hijos, solamente a la propia instancia y componentes hijos con contenido insertado en _slot_.
```

File: src/v2/api/index.md [hunk 10]
```diff
@@ -1739,99 +1739,99 @@ type: api
...
 ### vm.$nextTick( [callback] )
 
-- **Arguments:**
+- **Argumentos:**
   - `{Function} [callback]`
 
 - **Uso:**
 
-  Defer the callback to be executed after the next DOM update cycle. Use it immediately after you've changed some data to wait for the DOM update. This is the same as the global `Vue.nextTick`, except that the callback's `this` context is automatically bound to the instance calling this method.
+  Pospone la ejecución de la función _callback_ para luego del próximo ciclo de actualización del DOM. Utilize este método inmediatamente luego de modificar datos para esperar por la actualización del DOM. Este método es el mismo que el método global `Vue.nextTick`, excpeto que el contexto `this` de la función _callback_ es automáticamente vinculado a la instancia que invoco este método.
```
Comment (FLarra) [inline src/v2/api/index.md]: There is a misspelling here:

```suggestion
  Pospone la ejecución de la función _callback_ para luego del próximo ciclo de actualización del DOM. Utilice este método inmediatamente luego de modificar datos para esperar por la actualización del DOM. Este método es el mismo que el método global `Vue.nextTick`, excepto que el contexto `this` de la función _callback_ es automáticamente vinculado a la instancia que invoco este método.
```

File: src/v2/api/index.md [hunk 11]
```diff
@@ -1739,99 +1739,99 @@ type: api
...
 
-  > New in 2.1.0+: returns a Promise if no callback is provided and Promise is supported in the execution environment. Please note that Vue does not come with a Promise polyfill, so if you target browsers that don't support Promises natively (looking at you, IE), you will have to provide a polyfill yourself.
+  > Nuevo en 2.1.0+: retorna una promesa (_Promise_) si no fue provista una función _callback_ y estas son soportadas en el entorno de ejecución. Note que Vue no trae un _polyfill_ para promesas, por lo tanto, si su navegador no soporta promseas de forma nativa (mirándote, IE), usted deberá agregar un _polyfill_ por su cuenta.
```
Comment (FLarra) [inline src/v2/api/index.md]: There is a typo here:

```suggestion
  > Nuevo en 2.1.0+: retorna una promesa (_Promise_) si no fue provista una función _callback_ y estas son soportadas en el entorno de ejecución. Note que Vue no trae un _polyfill_ para promesas, por lo tanto, si su navegador no soporta promesas de forma nativa (mirándote, IE), usted deberá agregar un _polyfill_ por su cuenta.
```

File: src/v2/api/index.md [hunk 12]
```diff
@@ -1739,99 +1739,99 @@ type: api
...
       example: function () {
-        // modify data
+        // modificar datos
         this.message = 'changed'
-        // DOM is not updated yet
+        // el DOM no está actualizado todavía
         this.$nextTick(function () {
-          // DOM is now updated
-          // `this` is bound to the current instance
+          // el DOM está actualizaod
```
Comment (FLarra) [inline src/v2/api/index.md]: Typo :) 

```suggestion
          // el DOM está actualizado
```

File: src/v2/api/index.md [hunk 13]
```diff
@@ -1739,99 +1739,99 @@ type: api
...
   - [Vue.nextTick](#Vue-nextTick)
-  - [Async Update Queue](../guide/reactivity.html#Async-Update-Queue)
+  - [Cola de actualización asíncrona](../guide/reactivity.html#Cola-de-actualizacion-asincrona)
 
 ### vm.$destroy()
 
 - **Uso:**
 
-  Completely destroy a vm. Clean up its connections with other existing vms, unbind all its directives, turn off all event listeners.
+  Destruir completamente una `vm`. Limpia todas sus conecciones con otras `vms` existentes, desvincula todas sus directivas y apaga todas sus escuchas a eventos.
```
Comment (FLarra) [inline src/v2/api/index.md]: Misspelling here:

```suggestion
  Destruir completamente una `vm`. Limpia todas sus conexiones con otras `vms` existentes, desvincula todas sus directivas y apaga todas sus escuchas a eventos.
```

File: src/v2/api/index.md [hunk 14]
```diff
@@ -1739,99 +1739,99 @@ type: api
...
 
-  Triggers the `beforeDestroy` and `destroyed` hooks.
+  Dispará los `hooks` `beforeDestroy` y `destroyed`.
```
Comment (FLarra) [inline src/v2/api/index.md]: It's:

```suggestion
  Dispara los `hooks` `beforeDestroy` y `destroyed`.
```

**Review by FLarra — CHANGES_REQUESTED** [review]
Nice Job. Left you some comments

File: src/v2/api/index.md (hunk 5 above)
Comment (Tintef) [reply src/v2/api/index.md]: I like that idea! @FLarra 😄

File: src/v2/api/index.md (hunk 10 above)
Comment (Tintef) [reply src/v2/api/index.md]: Yeah, I always that mistake 🤦‍♂ I know it's wrong, but I still manage to spell it wrong 😅

File: src/v2/api/index.md (hunk 13 above)
Comment (Tintef) [reply src/v2/api/index.md]: Damn 😅

File: src/v2/api/index.md (hunk 7 above)
Comment (FLarra) [reply src/v2/api/index.md]: @Tintef you missed this typo + misspelling 😃

**Review by FLarra — APPROVED** [review]
There is only one comment you missed (look for my new comment to find where it is), apart from that I'm good with the PR. I'll approve 🚀

**Pull request merged**